openassessment/xblock/studio_mixin.py

```diff
@@ -81,14 +138,7 @@ def studio_view(self, context=None):
 
     def editor_context(self):
         """
-        Update the XBlock's XML.
-
-        Args:
-            data (dict): Data from the request; should have a value for the key 'xml'
-                containing the XML for this XBlock.
-
-        Keyword Arguments:
-            suffix (str): Not used
+        Gets the editor context, ready for rendering.
```

Comment:
This docstring was wildly out-of-date, should now more accurately reflect what's happening here